<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemGeneral</class>
 <widget class="QWidget" name="ItemGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scaleEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Stats</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Durability</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duraEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Identified</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rarity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Repair Durability Penalty</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repairDuraEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Level</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="levelEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="itemTypeCombo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Common</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rare</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Legendary</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Unique</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="identCheck">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>False</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
